x-pack/plugins/apm/server/routes/settings/agent_configuration/get_config_applied_to_agent_through_fleet.ts

```diff
@@ -7,14 +7,17 @@
 
 import { termQuery, rangeQuery } from '@kbn/observability-plugin/server';
 import datemath from '@kbn/datemath';
+import { APMIndices } from '@kbn/apm-data-access-plugin/server';
 import { METRICSET_NAME } from '../../../../common/es_fields/apm';
 import { APMInternalESClient } from '../../../lib/helpers/create_es_client/create_internal_es_client';
 
 export async function getConfigsAppliedToAgentsThroughFleet(
-  internalESClient: APMInternalESClient
+  internalESClient: APMInternalESClient,
+  apmIndices: APMIndices
 ) {
   const params = {
-    index: internalESClient.apmIndices.metric,
+    index: apmIndices.metric,
```

Comment:
why do you need to pass the `apmIndices` [here](https://github.com/elastic/kibana/pull/170733/files#diff-0cf35eb8288590cb5b0ba54ec6dff7a113bfcddfa3bd063300528c98896e6b11R29) from the resources  when you could load them from the  apm-data-access-plugin

```
import { APMIndices } from '@kbn/apm-data-access-plugin/server';
```

Reply:
`APMIndices` (that you mention) is a type. `apmIndices` is a value (an object).